apps/typegpu-docs/src/content/docs/integration/webgpu-interoperability.mdx

```diff
@@ -0,0 +1,327 @@
+---
+title: WebGPU Interoperability
+draft: true
+---
+
+TypeGPU is built in a way that allows you to pick and choose the primitives you need, incrementally adopt them into your project and have a working app at each step
+of the process. We go to extremes to ensure that turning even a single buffer into our typed variant improves the developer experience, and does not require changes
+to the rest of the codebase.
+
+The **non-contagious** nature of TypeGPU means that ejecting out in case raw WebGPU access is required can be done on a very granular level.
```

Comment:
```suggestion
The **non-contagious** nature of TypeGPU means that ejecting out, in case raw WebGPU access is required, can be done on a very granular level.
```